### PR TITLE
Component render setup

### LINKED ISSUE #2: Create App Component
As a dev I want to create a App presentational component which renders the following components:
Header
Form
CardArea
IDPage

### PR DESCRIPTION
**Pull request by timkeresey**
### What does this PR do?

- Fixes file structure for components
- declares components with their returned JSX elements
- imports components into necessary files.

### Where should the reviewer start?

- All component files in the src directory.

### How should this be manually tested?

- by observing the browser and using the dev tools

### What are the relevant tickets?

- closes #2 
- #3 , #4 , #6 , #9 , #12 , #16

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main>
+      <Header />
+      <IDForm />
+      <IDArea />
+      <Route path='/id-page'>
```
Comment (taylorjohnson141) [inline src/components/App/App.js:14]: Great start to Router

File: src/components/IDForm/IDForm.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import './IDForm.css';
+
+
+const IDForm = () => {
```
Comment (taylorjohnson141) [inline src/components/IDForm/IDForm.js:4]: Looks good! We'll have to import the reac-hooks-form. But the set up looks good!

**Review by ericberglund117 — APPROVED** [review]
Set up looks good. Now we have a solid structure and an idea where to start building out our actions and reducers

File: src/components/MissionArea/MissionArea.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import './MissionArea.css';
+
+const MissionArea = () => {
```
Comment (taylorjohnson141) [inline src/components/MissionArea/MissionArea.js:3]: I think we'll have to change the story names and mission names around or change the text but it looks good!

**Review by taylorjohnson141 — APPROVED** [review]

**Pull request merged**